src/SDDPoptimize.jl

```diff
@@ -52,8 +52,8 @@ function solve_SDDP(model::SPModel,
     end
 
     # Run SDDP upon example:
-    run_SDDP!(model, param, V, problems, display)
-    return V, problems
+    ic = run_SDDP!(model, param, V, problems, display)
+    return V, problems, ic
 end
```

Comment:
This is a good question. I think we need to think further about the user interface and the output of the main functions